project/core/urls.py

```diff
@@ -13,11 +13,13 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from re import template
 from django.conf import settings
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import include, path
 from django.views.generic.base import TemplateView
+from django.contrib.auth import views as auth_views
```

Comment:
I like that we are using core views, so we can rely on existing code instead of writing our own.